### PR TITLE
feat / add standard trading rule

### PR DESCRIPTION
**Pull request by yzhang1994**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**A description of the changes proposed in the pull request**:
- create standard trading rule class
- replace binance trading rule with the standard
- replace coinbase pro trading rule with the standard
- change binance test to not use LOOMETH trading pair (It's not wrong but it is unnecessary to require an additional pair for the cancel_all function)

### CODE REVIEW

File: hummingbot/market/binance/binance_market.pyx [hunk 1]
```diff
@@ -604,11 +561,39 @@ cdef class BinanceMarket(MarketBase):
             int64_t current_tick = <int64_t>(self._current_timestamp / 60.0)
         if current_tick > last_tick or len(self._trading_rules) < 1:
             exchange_info = await self.query_api(self._binance_client.get_exchange_info)
-            trading_rules_list = TradingRule.parse_exchange_info(exchange_info)
+            trading_rules_list = self._format_trading_rules(exchange_info)
             self._trading_rules.clear()
             for trading_rule in trading_rules_list:
                 self._trading_rules[trading_rule.symbol] = trading_rule
 
+    def _format_trading_rules(self, exchange_info_dict: Dict[str, Any]) -> List[TradingRule]:
```
Comment (chowryan) [inline hummingbot/market/binance/binance_market.pyx]: https://github.com/sammchardy/python-binance/blob/fd99e288e5863e59f2d90e261483a7941e96e5a7/binance/client.py#L234

File: hummingbot/market/coinbase_pro/coinbase_pro_market.pyx [hunk 2]
```diff
@@ -522,11 +483,26 @@ cdef class CoinbaseProMarket(MarketBase):
             int64_t current_tick = <int64_t>(self._current_timestamp / 60.0)
         if current_tick > last_tick or len(self._trading_rules) <= 0:
             product_info = await self._api_request("get", path_url="/products")
-            trading_rules_list = TradingRule.parse_exchange_info(product_info)
+            trading_rules_list = self._format_trading_rules(product_info)
             self._trading_rules.clear()
             for trading_rule in trading_rules_list:
                 self._trading_rules[trading_rule.symbol] = trading_rule
 
+    def _format_trading_rules(self, trading_rules: List[Any]) -> List[TradingRule]:
```
Comment (chowryan) [inline hummingbot/market/coinbase_pro/coinbase_pro_market.pyx]: rename trading_rules to raw_trading_rules?

Comment (yzhang1994) [reply hummingbot/market/coinbase_pro/coinbase_pro_market.pyx]: done

File: hummingbot/market/binance/binance_market.pyx (hunk 1 above)
Comment (yzhang1994) [reply hummingbot/market/binance/binance_market.pyx]: DONE

**Pull request merged**